test-app/app/views/layouts/application.html.erb

```diff
@@ -7,7 +7,7 @@
     <%= csp_meta_tag %>
 
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
-    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <%#= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
```

Comment:
Why comment?

Reply:
There was an error, I searched StackOverflow it said to delete that line but I wanted to leave it there.

Reply:
The web page loaded after that was commented out

Reply:
So, why leave this here? Why not just remove it altogether?

Reply:
cuz it should be there for the webpack to work. But because of the error, I had to comment it out for it to work for now. Just a temporary solution. Have to find a way to solve the error